src/agent/debugger/src/target.rs

```diff
@@ -616,18 +618,15 @@ impl Target {
     pub(crate) fn handle_single_step(&mut self, step_state: StepState) -> Result<()> {
         self.single_step.remove(&self.current_thread_handle);
 
-        match step_state {
-            StepState::Breakpoint { pc } => {
-                write_instruction_byte(self.process_handle, pc, 0xcc)?;
+        if let StepState::Breakpoint { pc } = step_state {
```

Comment:
Please revert this change - it will cause an annoying merge conflict for my PR.

Reply:
reverted